docs/peer_testing/peer_testing_1/Instructions.md

```diff
@@ -0,0 +1,45 @@
+# Peer Testing Instructions
+Examiner: Hello! Here are some instructions for peer testing. We have you complete a few basic tasks to test out our new
+language processor.
+
+### Starting Test for Rhubarb
+Examiner: First, please open `index.ts`. Navigate to `..\The-Speaking-Portal-Project\speaking-portal-project` and enter 
+`npm start`. If for any reason the file does not load immediately notify me, the examiner.
+
+### Missing File Test
+Examiner: Please set `language_selection = -1` in `index.txt`. Any nonexistent number will do. When you are ready, 
```

Comment:
What do you mean by nonexistent number?

Reply:
Any number which would not be in the set of options, not sure what wording to use there

Reply:
Ah, okay! It'll be clearer when the whole thing is in front of the person, then!